permissions/plugin-support-core.yml

```diff
@@ -8,4 +8,4 @@ developers:
 - "schristou"
 - "stephenconnolly"
 - "batmat"
-
+- "owood"
```

Comment:
I hope/suppose the parser isn't an old batshit and would fail without a newline...